app/src/Pages/Portal/Contributions/Utils/ContributionsFields.js

```diff
@@ -391,6 +392,35 @@ export const validate = (values) => {
       error.lastNameOrEntity = "Last name is required."
     }
   }
+  
+  if(( occupation == "Other" ) && !checkNoEmptyString(employerName)){
+    error.employerName = "Employer name is required."
+  }
+
+  if(( occupation == "Other" ) && !checkNoEmptyString(employerCity)){
+    error.employerCity = "Employer city is required."
+  }
+
+  if(( occupation == "Other" ) && !checkNoEmptyString(employerState)){
+    error.employerState = "Employer state is required."
+  }
 
+  if(( occupation == "Other" ) && !checkNoEmptyString(employerZipcode)){
+    error.employerZipcode = "Employer zipcode is required."
+  }
   return error
```

Comment:
@punkael here's another thought- what if we put in an `else` that specifically sets NOT required for those fields (essentially setting the validity of those fields to true)?

I haven't had luck yet with making that happen, but thought maybe the idea would kick something off in your brain that might help! 😁